fastlane/metadata/en-US/name.txt

```diff
@@ -1 +0,0 @@
-WooCommerce
```

Comment:
I double-checked with `diff -s fastlane/metadata/en-US fastlane/metadata/default` and confirmed that everything that was deleted from `en-US/` in this PR is already in `default/` with the exact same copy for each, so it's indeed safe (and expected, given the PR's changes and new tooling and approach) to delete all of them here, without needing to update `default/*.txt` files' contents, as they already match 👍